src/source/Sdp/Deserialize.c

```diff
@@ -1,37 +1,48 @@
-#define LOG_CLASS "SDP"
+#define LOG_CLASS "SDPDeserialize"
 #include "../Include_i.h"
+#include "sdp_deserializer.h"
 
-STATUS parseMediaName(PSessionDescription pSessionDescription, PCHAR pch, UINT32 lineLen)
+STATUS parseMediaName(PSessionDescription pSessionDescription, PCHAR mediaValue, size_t mediaValueLength)
 {
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
+    size_t copyLength;
```

Comment:
use `SIZE_T` macro instead of `size_t`

Reply:
Updated via commit 4711d2d9a8eaa0117ed8f63c7b8b957ac3a1d283.

Reply:
I got this compile error in [mac-os-build-clang](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/actions/runs/7927210884/job/21643215938?pr=1913) and [mac-os-build-gcc](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/actions/runs/7927210884/job/21643215315?pr=1913) if I set size_t to SIZE_T. Looks like they're set to different data structures in PIC repo.

```
2024-02-16T07:31:48.7756110Z /Users/runner/work/amazon-kinesis-video-streams-webrtc-sdk-c/amazon-kinesis-video-streams-webrtc-sdk-c/src/source/Sdp/Deserialize.c:111:82: error: incompatible pointer types passing 'SIZE_T *' (aka 'unsigned long long *') to parameter of type 'size_t *' (aka 'unsigned long *') [-Werror,-Wincompatible-pointer-types]
2024-02-16T07:31:48.7759080Z         sdpResult = SdpDeserializer_GetNext(&ctx, &type, (const CHAR**) &pValue, &valueLength);
2024-02-16T07:31:48.7760320Z                                                                                  ^~~~~~~~~~~~
2024-02-16T07:31:48.7762560Z /Users/runner/work/amazon-kinesis-video-streams-webrtc-sdk-c/amazon-kinesis-video-streams-webrtc-sdk-c/open-source/include/kvssdp/sdp_deserializer.h:22:47: note: passing argument to parameter 'pValueLength' here
2024-02-16T07:31:48.7764560Z                                      size_t * pValueLength );
2024-02-16T07:31:48.7765460Z                                               ^
2024-02-16T07:31:48.7784270Z 1 error generated.
```

Reply:
It passes with casting (cast to size_t). Let me know if you have other concern.

Reply:
It's still failing on [mac-os-build-clang](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/actions/runs/7927508659/job/21646981674) when using SIZE_T. Revert this back to size_t for now.